securedrop_client/gui/buttons.py

```diff
@@ -0,0 +1,40 @@
+"""
+SecureDrop Buttons
+
+These are appropriate for use in the SecureDrop Client. Besides that, they are regular Qt buttons.
```

Comment:
"Besides that, they are regular Qt buttons" isn't very clear to me. It would be helpful to elaborate more on which classes belong in this file, or just remove this sentence.

Reply:
Ok, that was meant as a hint to all the benefits that come from being a regular Qt thing, e.g. accessibility. Let me make that clearer!

Reply:
I feel like this and the more recent update is a little reactionary to our conversation about the "offline model" link, and I could see it being confusing to newcomers who don't understand what you mean by: 
"These buttons look and behave appropriately in the context of SecureDrop client,
but besides that, they are regular Qt buttons which means they benefit from
the full-range of accessibility features provided by the framework.
When adding or extending buttons in this file, please make sure the implementation
doesn't obstruct the default Qt buttons API."

It might be clearer to just say that classes added to this file should inherit from `QPushButton`.

Reply:
If you want to be opinionated about what a button is, this might be the right place to do it. For instance, you could say something about anything that has an onPressed event should be a button. You could also elaborate on the accessbility benefits of QPushButtons.

Reply:
> It might be clearer to just say that classes added to this file should inherit from `QPushButton`.

That's a very good point.